tests/unit_test/app/routes/test_git_tokens.py

```
@@ -0,0 +1,259 @@
+
+from fastapi import status
+
+from app.routes.git_tokens import mask_token
+
+
+class TestMaskToken:
+	"""Test cases for token masking functionality."""
+	
+	def test_mask_token_normal_length(self):
+		"""Test masking for normal length token."""
+		token = "ghp_1234567890abcdef"
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)